meta-luneui/recipes-webos/cmake-modules-webos-native/cmake-modules-webos/0002-_webos_init_install_vars-change-webos_testsdir-defau.patch

```diff
@@ -18,16 +17,17 @@ Only build tested.
 [PLAT-85921] Move /opt into /usr/opt
 
 Change-Id: I8d49d3cab0cec1d29a0ff7feea4d7b0fb66fbe9a
-Upstream-Status: Pending
 ---
+Upstream-Status: Submitted [http://gpro.lge.com/c/webosose/cmake-modules-webos/+/255719 _webos_init_install_vars: change webos_testsdir default value]
+
  webOS/webOS.cmake | 4 ++--
  1 file changed, 2 insertions(+), 2 deletions(-)
 
 diff --git a/webOS/webOS.cmake b/webOS/webOS.cmake
-index 37aa2ca..48ae256 100644
+index e72e7df..7119c72 100644
 --- a/webOS/webOS.cmake
 +++ b/webOS/webOS.cmake
-@@ -486,8 +486,8 @@ macro(_webos_init_install_vars)
+@@ -487,8 +487,8 @@ macro(_webos_init_install_vars)
  	# Deprecated name for WEBOS_INSTALL_MOUNTABLESTORAGEDIR
  	_webos_set_from_env(WEBOS_INSTALL_LOCALSTORAGEDIR           webos_mountablestoragedir       ${WEBOS_INSTALL_MOUNTABLESTORAGEDIR})
```

Comment:
```
Applying patch 0002-_webos_init_install_vars-change-webos_testsdir-defau.patch
patching file webOS/webOS.cmake
patch unexpectedly ends in middle of line
Hunk #1 succeeded at 491 with fuzz 1 (offset 4 lines).
```

Reply:
These patch files come from OSE directly, so should be present there too?

Reply:
It's not present in OSE, otherwise I would have noticed it there.

Reply:
Your version is missing the newline at the end:
```
luneos-nanbield $ diff -uNr meta-webos-ports/meta-luneui/recipes-webos/cmake-modules-webos-native/cmake-modules-webos/0002-_webos_init_install_vars-change-webos_testsdir-defau.patch /OE/lge/layers/meta-webos
ose/meta-webos/recipes-webos/cmake-modules-webos/cmake-modules-webos/0002-_webos_init_install_vars-change-webos_testsdir-defau.patch
--- meta-webos-ports/meta-luneui/recipes-webos/cmake-modules-webos-native/cmake-modules-webos/0002-_webos_init_install_vars-change-webos_testsdir-defau.patch   2023-09-28 10:43:12.597508671 +0200
+++ /OE/lge/layers/meta-webosose/meta-webos/recipes-webos/cmake-modules-webos/cmake-modules-webos/0002-_webos_init_install_vars-change-webos_testsdir-defau.patch       2023-08-29 01:15:17.333582986 +0200
@@ -37,4 +37,4 @@
 +      _webos_set_from_env(WEBOS_INSTALL_TESTSDIR                  webos_testsdir                  ${WEBOS_INSTALL_PREFIX}/opt/webos/tests)
  
        # This tree contains subdirectories of various types of customization data
-       _webos_set_from_env(WEBOS_INSTALL_WEBOS_CUSTOMIZATIONDIR              webos_customizationdir                  ${WEBOS_INSTALL_PREFIX}/palm/customization)
\ No newline at end of file
+       _webos_set_from_env(WEBOS_INSTALL_WEBOS_CUSTOMIZATIONDIR              webos_customizationdir                  ${WEBOS_INSTALL_PREFIX}/palm/customization)
```